### DISCUSSION TITLE
Pro board?

### DISCUSSION THREAD

**Discussion by stevempotter** [phil-barrett/grblHAL-teensy-4.x#43] [Q&A]
Hi Phil,
Terje mentioned a "Pro board" in discussion #27. I did not see that on Tindie. What is different about it and where is it available? Can I see a schematic of it somewhere to help interpret his comments?

**Comment by phil-barrett** [comment]
I have been slow getting it available.  Sigh.  I am currently out of town, back on the  19th. It is similar to the current board with some upgrades - isolated spindle output, pluggable screw terminals,  12v switches and sensors. A number of smaller improvements. The info is easily accessible on my home PC, not so easy from my phone. I'll try to dig up a link for you.
